client/src/app/core/components/login-dialog/login-dialog.component.html

```diff
@@ -0,0 +1,12 @@
+
+
+<div class="login-container">
+  <mat-form-field>
```

Comment:
Where are the form controls and form bindings? 
No submit action so far? It seems like the login and signup aren't completed

Reply:
@mehdirajani5 I see this file isn't being used. **Please remove this**